test/unit/nedap/utils/test/api.cljc

```diff
@@ -146,6 +147,15 @@
                   :to 1
                   :with =)))
 
+  (testing ":with match?"
```

Comment:
One change that you may want to add is with a nested matcher, say `matchers/in-any-order`

```clojure
(testing "with match? and matcher"
  (letfn [(db-insert [db val] 
            (swap! db (fn [vs] (shuffle (conj vs v)))))]
    (let [db (atom (shuffle [:a :b :c :d]))]
      (sut/expect (swap! db db-insert :d)
        :to-change @db
        :from (matchers/in-any-order [:a :b :c :d])
        :to (matchers/in-any-order [:a :b :c :d :e])
        :with match?))))
```

(Or some variation, right)

Reply:
The existing test already highlighted matchers/embeds (implicitly though). Added a `matchers/in-any-order` one too, although without `shuffle` to make sure no collision occurs (e.g. `(match? (in-any-order [1 2 3] [1 2 3]))`).

Thanks!

Reply:
Thats cool; I just wanted to show that using `match?` as a `:with` is as flexible as using `(is (match? ... ...))` is in the test, and so you did :).